pyanaconda/modules/storage/platform.py

```diff
@@ -187,6 +204,14 @@ def stage1_suggestion(self):
             "disk, mounted at /boot/efi."
         )
 
+    @property
+    def stage1_descriptions(self):
+        """The platform-specific descriptions of the stage1 device."""
+        return {
+            "partition": _("Apple EFI Boot Partition"),
```

Comment:
Please create constants for all these strings. I know it will be many but right now it's not consistent and it's pretty easy to miss the constant above.

Reply:
Fixed.